### PR TITLE
chore: support auto release

### PR DESCRIPTION
**Pull request by pddpd**
### PR includes

1. Add semantic-release to support auto release.
2. Add `auto-release.yml` workflow to run auto release and after stable brach updated.
3. Add `release-notification.yml` workflow to give the DingTalk group a notice after release.
4. Add `sync-packgejson-and-changelog-after-auto-release.yml` workflow to update master branch after auto release.

### CODE REVIEW

**Comment by pddpd** [comment]
Later, there will only be the stable branch. The alpha branch is added here to test whether workflows work.

File: .github/workflows/sync-packgejson-and-changelog-after-auto-release.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: 🔁 Sync Package.json and CHANGELOG After Auto Release
```
Comment (neoddish) [inline .github/workflows/sync-packgejson-and-changelog-after-auto-release.yml:1]: the name of this .yml file is way to long...and there is a typo 'packge'.

just 'after-release' and leave these desc as a comment in this file.

File: .github/workflows/auto-release.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        run: npm install
+
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.RELEASE_GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+        run: npm run release
+
+      # If failed, give the DingTalk group a notice
+      - name: Release failed giving the DingTalk group a notice
```
Comment (neoddish) [inline .github/workflows/auto-release.yml:32]: wow

File: .github/workflows/sync-packgejson-and-changelog-after-auto-release.yml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        run: |
+          git checkout -b chore-sync
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Push
+        run: |
+          git add .
+          git commit -m "chore: 🤖 update package.json and CHANGELOG" --no-verify
```
Comment (neoddish) [inline .github/workflows/sync-packgejson-and-changelog-after-auto-release.yml:46]: hmmm...personally, I don't like the emoji in git commits...

File: .github/workflows/sync-packgejson-and-changelog-after-auto-release.yml (hunk 1 above)
Comment (pddpd) [reply .github/workflows/sync-packgejson-and-changelog-after-auto-release.yml]: changed to `after-auto-release`

File: .github/workflows/sync-packgejson-and-changelog-after-auto-release.yml (hunk 3 above)
Comment (pddpd) [reply .github/workflows/sync-packgejson-and-changelog-after-auto-release.yml]: done

**Comment by pddpd** [comment]
I've pushed new auto release codes here. Plz note that after merging this PR, do not merge codes to `stable` temporarily. I need to add commits with tags in the next PR to ensure auto release runs correctly.

**Review by neoddish — APPROVED** [review]
LG

**Pull request merged**